File: Js/Fibonacci.js
function calcularFibonacci(n) {
    let primeiroTermo = 0;
    let segundoTermo = 1;
    console.log(primeiroTermo + " " + segundoTermo);

    for (let i = 2; i < n; i++) {
        let proximoTermo = primeiroTermo + segundoTermo;
        console.log(proximoTermo + " ");
        primeiroTermo = segundoTermo;
        segundoTermo = proximoTermo;
    }
}

const n = 8; // Substitua pelo número de termos da sequência que deseja calcular
calcularFibonacci(n);

File: Java/Fibonacci.java
package Java;
public class Fibonacci {
    public static void calcularFibonacci(int n) {
        int primeiroTermo = 0;
        int segundoTermo = 1;
        System.out.print(primeiroTermo + " " + segundoTermo + " ");

        for (int i = 2; i < n; i++) {
            int proximoTermo = primeiroTermo + segundoTermo;
            System.out.print(proximoTermo + " ");
            primeiroTermo = segundoTermo;
            segundoTermo = proximoTermo;
        }
    }

    public static void main(String[] args) {
        int n = 8; // Substitua pelo número de termos da sequência que deseja calcular
        calcularFibonacci(n);
    }
}
